http/headers/cache-control.json

```
@@ -56,7 +56,8 @@
             "description": "<code>immutable</code>",
             "support": {
               "chrome": {
-                "version_added": false
+                "version_added": false,
+                "notes": "See <a href='https://crbug.com/611416'>bug 611416</a>."
```

Given the bug is wontfix, maybe it would be good to expand on this a little, like:

```suggestion
                "notes": "There is no plan to implement this directive. See <a href='https://crbug.com/611416'>bug 611416</a>."
```